.clang-format

```diff
@@ -0,0 +1,28 @@
+PointerAlignment: Left
+ColumnLimit: 120
+BreakBeforeBraces: Allman
+IndentWidth: 4
+BinPackArguments: false
+BinPackParameters: false
+AllowShortFunctionsOnASingleLine: None
+Cpp11BracedListStyle: false
+AlwaysBreakAfterReturnType: None
+AlwaysBreakAfterDefinitionReturnType: None
+PenaltyReturnTypeOnItsOwnLine: 1000000
+BreakConstructorInitializers: BeforeComma
+BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterClass: false
+  AfterEnum: false
+  AfterFunction: true
+  AfterNamespace: false
+  AfterStruct: false
+  AfterUnion: false
+  AfterExternBlock: false
+  BeforeCatch: false
+  BeforeElse: false
+  SplitEmptyFunction: false
+  SplitEmptyRecord: false
+  SplitEmptyNamespace: false
+AllowShortFunctionsOnASingleLine : Inline
+SortIncludes: false
```

Comment:
The crypto team might have made edits to this during their Mexico offsite. Do we want to align with their views?

Reply:
Yes. This diff shouldn't be in this PR. Something weird is going on with PRs in this repo....